TwoPaneLayout/library/src/main/java/com/microsoft/device/dualscreen/twopanelayout/twopanelayout/TwoPaneLayoutCore.kt

```diff
@@ -29,6 +42,8 @@ internal fun SinglePaneContainer(
     pane2: @Composable TwoPaneScope.() -> Unit
 ) {
     currentSinglePane = Screen.Pane1.route // always start from Pane1 to maintain the expected backstack
+    isSinglePane = true
```

Comment:
Will it be better to check `isSinglePane` outside the container, based on `val isSinglePane = isSinglePaneLayout(windowState.windowMode, paneMode)`, instead of maintain the value between the two containers?

Reply:
great point! updated this, let me know if you think the new solution looks good